cmd/geth/snapshot.go

```diff
@@ -149,6 +165,68 @@ It's also usable without snapshot enabled.
 	}
 )
 
+func pruneBlock(ctx *cli.Context) error {
+	stack, config := makeConfigNode(ctx)
+	defer stack.Close()
+
+	chaindb := utils.MakeChainDatabaseForBlockPrune(ctx, stack, false)
+	chaindb.Close()
```

Comment:
Are you meaning ` defer chaindb.Close()`?

Reply:
No,  this is for closing it on purpose, because in the later code, there'll be many times of open/close the same db, crash will occur if did not close it.